assets/javascripts/kitten/components/carousel/simple-carousel.js

```diff
@@ -100,6 +102,32 @@ export class SimpleCarousel extends Component {
     this.setState({ currentPageNumber: numPage })
   }
 
+  handleKeyDown = event => {
+    if (event.key === 'ArrowLeft' || event.key === 'ArrowRight') {
```

Comment:
Je te propose plutôt de faire :
```suggestion
    if (['ArrowLeft', 'ArrowRight'].indexOf(event.key) !== -1) {
```